lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php

```diff
@@ -24,6 +25,8 @@
 /**
  * Command to execute DQL queries in a given EntityManager.
  *
+ * @deprecated The DQL command is deprecated and will be removed in Doctrine ORM 3.0.
```

Comment:
```suggestion
 * @deprecated The DQL command is deprecated and will be removed without replacement in Doctrine ORM 3.0.
```

for the case that someone reads through the code and ignored UPGRADE.md